features/step_definitions/functional_tests/functional_tests_steps.rb

```diff
@@ -55,18 +93,35 @@
         "test_instances",         # Requires the "first" network returned to be
         "test_servers"            # non-external
       ]
+    when "python-manilaclient"
+      [
+        "test_create_delete_share_type",                   # uses DHSS True (currently not default)
+        "test_list_with_debug_flag",                       # Similar to test_lists except for debug flag
+        "test_scheduler_stats",                            # Requires a backend named "mybackend" 
+        "test_quotas",                                     # uses assertRaises but command does not fail. Needs correction upstream.
```

Comment:
Metrics/LineLength: Line is too long. [132/100] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)